### PR TITLE
Update site to v5.2 of the kit

### PR DESCRIPTION
**Pull request by vickytnz**
Site updated with files for v5.2

This now makes it work on Codespaces, though i have noticed that it seems to make it run on port 3001 rather than 3000 which may need documenting somewhere (or at least some troubleshooting in case it's something to do with my setup).

### CODE REVIEW

File: lib/example-templates/blank-transactional.html
Comment (frankieroberto) [inline lib/example-templates/blank-transactional.html:1]: I don't think we should add all these example templates back into the docs site, as that’ll just confuse us in future – they should remain in the prototype kit repo only.

File: lib/prototype-admin/password.html
Comment (frankieroberto) [inline lib/prototype-admin/password.html:1]: This file isn't needed for the docs site as there’s not authentication needed.

**Review by frankieroberto — CHANGES_REQUESTED** [review]
I'm not sure it makes sense to copy all these changes from the prototype kit back into the docs site. Whilst the 2 repos started off the same, we’ve been teasing them apart.

The docs site doesn't really need to 'use' the prototype kit itself - it can just be a simple Express site which uses NHS Frontend in a similar way.

It could even be switched to using a static site generator framework instead like Eleventy or Jekyll or Metalsmith or Hugo - which would be cheaper to host. But currently we do have a single example of a dynamic page (the [branching example](https://prototype-kit.service-manual.nhs.uk/examples/branching)) which prevents it - and @edwardhorsford did suggest that we could add some more dynamic examples (eg validation?) in future.

It might be worth closing this pull request and re-opening smaller ones with any of the useful changes from the NHS prototype kit that we might want to copy across (eg `devcontainer.json`)?

**Comment by vickytnz** [comment]
> I'm not sure it makes sense to copy all these changes from the prototype kit back into the docs site. Whilst the 2 repos started off the same, we’ve been teasing them apart.
> 
> The docs site doesn't really need to 'use' the prototype kit itself - it can just be a simple Express site which uses NHS Frontend in a similar way.
> 
> It could even be switched to using a static site generator framework instead like Eleventy or Jekyll or Metalsmith or Hugo - which would be cheaper to host. But currently we do have a single example of a dynamic page (the [branching example](https://prototype-kit.service-manual.nhs.uk/examples/branching)) which prevents it - and @edwardhorsford did suggest that we could add some more dynamic examples (eg validation?) in future.
> 
> It might be worth closing this pull request and re-opening smaller ones with any of the useful changes from the NHS prototype kit that we might want to copy across (eg `devcontainer.json`)?

That's fair - though I'd like that documented somewhere. Interestingly I think the branching examples should go as tehy're tricky to maintain and in the govuk version are so buried that I didn't know they existed (I'd also say inconsistent givne we don't have code examples of the basic tutorial). 

Strictly speaking we don't need codespaces for the docs site, it was more that Paul's pull request had gone stale and so out of curiosity I wanted to try codespaces only for everything to just break.

**Pull request closed without merging**